docs/explanation/algorithm-overview.md

```diff
@@ -12,10 +12,16 @@ Input Image → Preprocessing → Localization → Decoding → Result
 
 ### 1. Preprocessing {#preprocessing}
 
+**Built-in preprocessing:**
```

Comment:
isn't there a built-in config that also preprocesses down to a single channel?  or does that just *act upon* only one channel, rather than altering the image wrapper?